templates/config.json

```diff
@@ -637,6 +695,11 @@
       "provider": "",
       "summary": "Management and each individual department/team holds regular company-wide / departmental / team meetings to review and discuss various aspects of business performance and objectives.",
       "guidance": "It is important to keep everyone on team / in the company regularly updated on the overall business strategy, goals and performance. One example is weekly/monthly roundtable or townhalls. At the individual team level, this could be the daily development standups/scrum sessions. This helps fosture an engaging, transparent and fast-moving culture.",
+      "instructionsSummary": "Confirm/tailor company-wide roundtable frequency.",
+      "instructions": [
+        "What tool do you use for HIPPAA Awareness Training?",
+        "What interval do employees to have to complete training?"
+      ],
```

Comment:
"How frequently do you hold company-wide roundtables to communicate updates?"

Reply:
this question is in cp-internal-comms